veg2hab/io/arcgis.py

```diff
@@ -351,6 +352,36 @@ def update_parameters(cls, parameters: List["arcpy.Parameter"]) -> None:
             params_dict[f"override_{idx}_geometry"].enabled = is_override_set
             params_dict[f"override_{idx}_truth_value_outside"].enabled = is_override_set
 
+            if not is_override_set:
+                params_dict[f"override_{idx}_truth_value"].value = None
+                params_dict[f"override_{idx}_geometry"].value = None
+                params_dict[f"override_{idx}_truth_value_outside"].value = None
+
+        # shift override criteria down, if one of 'm not set properly
+        for idx in range(1, MAX_N_OVERRIDE - 1):
+            if (
+                cls._is_override_empty(params_dict[f"override_{idx}_mits"].valueAsText)
+                and not cls._is_override_empty(params_dict[f"override_{idx + 1}_mits"].valueAsText)
+            ):
```

Comment:
Ik neem aan dat update_parameters bij iedere wijziging draait? Dat er dus geen gekke dingen gebeuren als je 2 op een volgende leeg hebt (omdat dat niet kan want t draait bij iedere wijziging)?

Reply:
Dat zou wel moeten ja. Ik kan eventueel de for loop vervangen door een while loop, zal ik dat doen?